vms/avm/vm_test.go

```diff
@@ -547,3 +547,159 @@ func TestGenesisGetUTXOs(t *testing.T) {
 		t.Fatalf("Wrong number of utxos (%d) returned", len(utxos))
 	}
 }
+
+func TestIssueDependentTx(t *testing.T) {
```

Comment:
Can you add a comment specifying what this is testing?

Reply:
Updated